app/src/main/res/values/strings.xml

```diff
@@ -1,16 +1,273 @@
+    <string name="settings_amount_of_days_to_offset_event_schedule_by">Amount of days to offset the event schedule by</string>
+    <string name="settings_schedule_events_in_5_minutes">Schedule notifications in 5 minutes instead of event time</string>
+    <!-- Settings end -->
+
+    <!-- Action Bar -->
+    <string name="action_report_bug">Report a Bug</string>
+    <string name="action_report_feedback">Feedback</string>
+    <string name="action_get_help">Get help</string>
+    <!-- Action Bar end -->
+
+    <!-- Event  -->
+    <string name="event">Event</string>
+    <string name="events">Events</string>
+    <string name="event_schedule">Event Schedule</string>
+    <string name="event_tracks">Event tracks</string>
+    <string name="event_weekday_from_to">%1$s from %2$s to %3$s</string>
+    <string name="event_unable_to_locate_room">Unable to locate room!</string>
+    <string name="event_run_by">Run by %1$s</string>
+    <string name="event_pager_search_by_title">Search by title</string>
+    <string name="event_upcoming">Upcoming events</string>
+    <string name="event_running">Running events</string>
+    <string name="event_favorited">Favorited events</string>
+    <string name="event_options_for">Event options for event %1$s</string>
+    <string name="event_add_to_favorites">Add event to favorites</string>
+    <string name="event_remove_from_favorites">Remove event from favorites</string>
+    <string name="event_changed_status">Changed event status!</string>
+    <string name="event_share_event">Share Event</string>
+    <string name="event_share_export_to_calendar">Export to calendar</string>
+    <string name="event_check_out_url">Check out %1$s!\n%2$s</string>
+    <string name="event_countdown_minutes">%1$d min</string>
+    <string name="event_did_not_find">I\'m sorry, we didn\'t find any event!</string>
+    <!-- Event end -->
+
+    <!-- Announcement -->
+    <string name="announcements">Announcements</string>
+    <string name="announcement_title">Announcement Title</string>
+    <string name="announcement_content">Announcement Content</string>
+    <string name="announcement_loading_failed">Unable to load announcement</string>
+    <!-- Announcement end -->
+
+    <!-- Dealer -->
+    <string name="dealers">Dealers</string>
+    <string name="dealer_name">Dealer Name</string>
+    <string name="dealer_subname">Subname</string>
+    <string name="dealer_about_artist">About the Artist</string>
+    <string name="dealer_about_art">About the Art</string>
+    <string name="dealer_telegram_handle">%1$s @ Telegram</string>
+    <string name="dealer_twitter_handle">%1$s @ Twitter</string>
+    <string name="dealer_location_and_availability">Location &amp; Availability</string>
+    <string name="dealer_only_present_on">{fa-exclamation-triangle 24sp} Only present on: %1$s</string>
+    <string name="dealer_located_in_the_after_dark">{fa-moon-o 24sp}  Located in the After Dark Dealers Den</string>
```

Comment:
(Nitpick) IMHO the FA icons shouldn't be part of the translation.

Reply:
Okay make sense, I'll fix these later when I'm at home